<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stack">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>1000</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>600</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="menu">
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>0</height>
      </size>
     </property>
     <widget class="QPushButton" name="search">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Search Brother</string>
      </property>
     </widget>
     <widget class="QPushButton" name="classes">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>425</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Classes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="list">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>500</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>List Brothers</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tree">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>425</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Brother's Tree</string>
      </property>
     </widget>
     <widget class="QPushButton" name="line">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>500</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Brother's Line</string>
      </property>
     </widget>
     <widget class="QPushButton" name="family">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>350</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Family Tree</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="brotherDir">
     <widget class="QLineEdit" name="searchBar">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>500</y>
        <width>251</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>500</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTableView" name="broList">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>25</y>
        <width>900</width>
        <height>450</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="listDir">
     <widget class="QTableView" name="broDir">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>25</y>
        <width>900</width>
        <height>475</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="back1">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>510</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="familyDir">
     <widget class="QPushButton" name="back2">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>510</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="treeDir">
     <widget class="QPushButton" name="back3">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>510</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lineDir">
     <widget class="QPushButton" name="back4">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>510</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
